### ISSUE TITLE
[Feature Request] Folder view/option for filler or channel bumps 

### ISSUE THREAD

**Issue by gils001** [ErsatzTV/ErsatzTV#1737]
![Screenshot (98)](https://github.com/ErsatzTV/ErsatzTV/assets/63884407/52ad3848-3f44-4817-a00e-71aaa281f1b3)

Hello,

I'm new to ErsatzTV. I previously used the PseudoTV Kodi addon. So far, I love the app. I'm sure there's something I'm doing incorrectly here, but I'd like to have a folder view or option when organizing and applying fillers. I do not require metadata or poster views for what are essentially commercials.

Currently, I'm adding filler via the 'Media Sources' tab, selecting 'Local Libraries' (I keep fillers for each channel within its own folder), assigning the media type to 'Other Videos', then to a 'Collection', and finally to the 'Filler Presets' tab. This can become extremely hectic when I add multiple 'Other Videos' folders for each channel. It feels like one too many steps to have ErsatzTV grab an item from a folder and add it to the schedule as a filler. I'm having a very tough time. Even if the solution is to sort the 'Other Videos' view by library name, it would help immensely when I add those fillers to a collection and setup a filler preset.

If I'm doing something wrong in the setup, please let me know. I'm still learning the ins and outs of the app.

Thank You

**Comment by jasongdove** [comment]
I agree the other video view is not ideal for that content type. I'll think about how we could improve it. Perhaps something closer to the image library view.

**Comment by jasongdove** [comment]
I am cleaning up GitHub issues and migrating feature requests to the ErsatzTV Fider instance.

This has moved to https://features.ersatztv.org/posts/17/folder-view-for-other-videos-library
